Dockerfile

```
@@ -0,0 +1,19 @@
+# Stage 1: Build the application
+FROM node:20.19 AS build
+
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm install
+
+COPY . .
+RUN ls -la && npm run build --prod
```

What's the purpose of the "RUN ls -la" here? 
If it was only for debugging purposes while creating this Dockerfile, I would suggest removing it.

```suggestion
npm run build --prod
```